udf_js/unzip.sql

```diff
@@ -0,0 +1,53 @@
+/*
+
+Unzips a GZIP string.   
+
+This implementation relies on the zlib.js library (https://github.com/imaya/zlib.js) and 
+the atob function for decoding base64.
+
+*/
+
+
+CREATE TEMP FUNCTION
+  udf_js_unzip (input BYTES)
```

Comment:
The `input` bytes are base64 encoded. I had to encode them in the UDF in order to decompress them.